esx_service/utils/kvESX.py

```diff
@@ -256,32 +257,42 @@ def load(volpath):
     """
     Load and return dictionary from the sidecar
     """
+    logging.info("Load dictionary from sidecar for vol path %s", volpath)
```

Comment:
#1419 has been filed to track removing the log added in this PR after the root cause of #1371 is found.